tests/sim/main.rs

```diff
@@ -9,15 +11,20 @@ use near_sdk_sim::ContractAccount;
 use near_sdk_sim::UserAccount;
 use near_sdk_sim::{call, deploy, lazy_static_include, to_yocto};
 use std::convert::TryInto;
+use std::str;
 
 near_sdk::setup_alloc!();
 
 const CONTRACT_ID: &str = "request_proxy";
+const FUNGIBLE_CONTRACT_ID: &str = "fungible_request_proxy";
```

Comment:
```suggestion
const PROXY_ID: &str = "request_proxy";
const FUNGIBLE_PROXY_ID: &str = "fungible_request_proxy";
```
To avoid the confusion with fungible token contract ID.